README.md

```diff
@@ -233,13 +233,154 @@ mstorage.info("<media_id>", new CompletionHandler<MediaInfo>() {
 });
 ```
 
-### Get media metadata
+### Get all media metadata
```

Comment:
I thought it'd be better to include all the methods which get, attach, or delete  all or some of metadata in one section.
For example; 
### Get media metadata 
* All
* Exif
* Google Photo Sphere XMP
* User metadata
* Specified user metadata

### Attach user metadata to media

### Delete specified user metadata from media
* User metadata
* Specified user metadata